driver-core/src/test/java/com/datastax/driver/core/policies/IdempotenceAwareRetryPolicyIntegrationTest.java

```diff
@@ -34,6 +34,7 @@
 /**
  * Integration test with an IdempotenceAwareRetryPolicy.
  */
+@SuppressWarnings("deprecation")
 public class IdempotenceAwareRetryPolicyIntegrationTest extends AbstractRetryPolicyIntegrationTest {
```

Comment:
I don't get any deprecation warnings here on 3.0.x. They do happen in 3.1.x though, the policy was deprecated (should we still test it?).